build.fsx

```diff
@@ -266,14 +272,14 @@ Target "CodeGen.NetCore" (fun _ ->
 )
 
 Target "Build.NetCore" (fun _ ->
-    [ "src/fsharp/FSharp.Compiler.Service/";
-      "src/fsharp/FSharp.Compiler.Service.ProjectCracker/";
-      "src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool/" ]
-    |> List.iter (run "dotnet" "-v pack %s -c Release")
+    [ "src/fsharp/FSharp.Compiler.Service.netcore/";
+      "src/fsharp/FSharp.Compiler.Service.ProjectCracker.netcore/";
+      "src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool.netcore/" ]
+    |> List.iter (run "dotnet" "pack %s -v detailed -c Release")
 )
 
 Target "RunTests.NetCore" (fun _ ->
-    runCmdIn "tests/service/" "dotnet" "test -c Release --result:TestResults.NetCore.xml;format=nunit3"
+    runCmdIn "tests/service.netcore/" "dotnet" "run -c Release -- --result:TestResults.NetCore.xml;format=nunit3"
```

Comment:
you can also specify the `-p` isntead of change working directory (it's easier to read)

`dotnet run -p tests/service.netcore/service.netcore.fsproj -c Release ...`